bundles/org.openhab.core.semantics/model/SemanticTags.csv

```diff
@@ -199,22 +203,22 @@ Equipment,AccentLight,LightSource,Accent Light,,
 Equipment,Lamp,LightSource,Lamp,,
 Equipment,FloodLight,LightSource,Flood Light,,
 Equipment,SpotLight,LightSource,Spot Light,,
-Equipment,WirelessAccessPoint,NetworkAppliance,Wireless Access Point,"AccessPoint, WAP, WiFi",
+Equipment,WirelessAccessPoint,NetworkAppliance,Wireless Access Point,"Access Point, WAP, WiFi, WiFi Access Point",
 Equipment,NetworkSwitch,NetworkAppliance,Network Switch,,
 Equipment,Router,NetworkAppliance,Router,,
 Equipment,Firewall,NetworkAppliance,Firewall,,
 Equipment,PetCare,,Pet Care,,
 Equipment,PetFeeder,PetCare,Pet Feeder,,
-Equipment,CatFlap,PetCare,Cat Flap,,
-Equipment,Aquarium,PetCare,Aquarium,,
+Equipment,PetFlap,PetCare,Pet Flap,"Cat Flap, Dog Flap",
+Equipment,Aquarium,PetCare,Aquarium,Fish Tank,
 Equipment,AudioVisual,,Audio Visual,,
 Equipment,MediaPlayer,AudioVisual,Media Player,,
-Equipment,WaterFeature,Pump,Water Feature,"Water fall, Pond pump",
+Equipment,WaterFeature,Pump,Water Feature,"Waterfall, Pond Pump",
 Equipment,Printer,,Printer,,
```

Comment:
Since I think there are many of us who have 3D printers, it might be a good idea to add it. We now have a binding for Bambulab too, so this would be immediately useful.
```
Equipment,3DPrinter,Printer,3D Printer,,
```

Reply:
but wait.... can Tags start with a number? I think not.

Reply:
Either `ThreeDPrinter` or `Printer3D` - the Label can be `3D Printer`